owner-onboarding-server/src/ov_management/mod.rs

```diff
@@ -0,0 +1,219 @@
+use std::sync::Arc;
+use openssl::pkey::{Private, PKey};
+use warp::{Filter};
+use std::{usize,str::FromStr};
+use fdo_data_formats::{ownershipvoucher::OwnershipVoucher, types::Guid};
+use serde::{Serialize, Deserialize};
+use warp::{hyper::StatusCode, http::HeaderValue, Buf, reply::Json};
+use crate::OwnerServiceUD;
+
+pub(crate) fn ov_filter(user_data:Arc<OwnerServiceUD>,owner_key: PKey<Private>) 
```

Comment:
Why are you passing in the private key here, instead of the public key corresponding to it?
Passing the private key around to code that doesn't need it is an unnecessary risk.